src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -263,7 +263,12 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
     // The source read may not have the RX tag, and if so, we ignore them through
     // the use of .flatMap and .get, with the latter returning an `Some(bases)` if the
     // UMI bases are present, `None` otherwise.
-    reads.flatMap(_.sam).flatMap(_.get[String](ConsensusTags.UmiBases))
+    reads.flatMap(_.sam).flatMap { rec =>
+      rec.get[String](ConsensusTags.UmiBases).map { umi =>
+        // when combining umi bases across multiple strands, we will need to reverse one or the other.
+        if (rec.firstOfPair) umi else umi.split('-').reverse.mkString("-")
```

Comment:
Having looked through things a bunch I think this makes sense.  However, I was initially worried because I thought this method had much broader usage.  I think the code fix is right, but I also think we should:

1. Rename this method to `toUmiBasesForConsensusUmiCalling()`
2. Add some explanation of how and why this method works

Because if we tried to use this elsewhere to just grab and examine the UMIs with the expectation that they'd conform to the usual duplex layout we'd make a big mess.